port/win/port_win.h

```diff
@@ -81,8 +81,8 @@ namespace rocksdb {
 
 namespace port {
 
-// VS 15
-#if (defined _MSC_VER) && (_MSC_VER >= 1900)
+// VS 15 / MinGW
+#if !defined(_MSC_VER) || (_MSC_VER >= 1900)
```

Comment:
It's equivalent; this checks for MSVC. Is this change really worth more CI cycles?